### PR TITLE
istio DNS

### PR DESCRIPTION
**Pull request by EItanya**

### CODE REVIEW

File: ci/setup-funcs.sh [hunk 1]
```diff
@@ -230,6 +221,25 @@ data:
         forward . ${ISTIO_COREDNS}:53
     }
 EOF
+}
+
+function install_istio() {
```
Comment (jenshu) [inline ci/setup-funcs.sh:226]: could you also update `pkg/meshctl/commands/demo/common/initialize/scripts/install_istio.sh` with these changes?

Comment (EItanya) [reply ci/setup-funcs.sh:226]: Is this something that we can consolidate in the future?

Comment (jenshu) [reply ci/setup-funcs.sh:226]: yep, we have this issue filed https://github.com/solo-io/gloo-mesh/issues/876

File: ci/setup-funcs.sh [hunk 2]
```diff
@@ -177,27 +180,15 @@ spec:
   values:
     global:
       pilotCertProvider: kubernetes
-      podDNSSearchNamespaces:
-      - global
 EOF
 }
 
-function install_istio() {
+# Operator spec for istio 1.8.x
```
Comment (harveyxia) [inline ci/setup-funcs.sh:186]:
```suggestion
# updates the kube-system/coredns configmap in order to resolve hostnames with a ".global" suffix, needed for istio < 1.8
```

File: test/e2e/istio/pkg/tests/failover_service.go [hunk 3]
```diff
@@ -24,7 +26,18 @@ func FailoverServiceTest() {
 		}
 	)
 
+	BeforeEach(func() {
```
Comment (harveyxia) [inline test/e2e/istio/pkg/tests/failover_service.go:29]: I'm inclined to not add code to deal with upstream bugs and instead just be aware of it internally—we've already disabled the status checks on 1.8.

What do others think? @joekelley @ilackarms

Comment (EItanya) [reply test/e2e/istio/pkg/tests/failover_service.go:29]: I did this as a result of a conversation with @ilackarms. I have no strong feeling either way.

File: docs/content/guides/installing_istio.md [hunk 4]
```diff
@@ -264,7 +266,14 @@ istiocoredns-7ffc9b7fcf-crhr2           2/2     Running   0          5d23h
 istiod-7795ccf9dc-vr4cq                 1/1     Running   0          5d22h
 ```
 
-We also have to enable Istio DNS for the `.global` stub domain for when we want to use multicluster communication. The following two blocks will enable Istio DNS for both clusters.
+{{% notice note %}}
+The following section of the doc describes how to modify `coredns` to enable Istio DNS for the `.global` stub domain. This is not necessary when running Istio > 1.8.x
+with Istio's new Smart DNS enabled. More information on this new DNS can be found in the following [blog post](https://istio.io/latest/blog/2020/dns-proxy/).
+If you have installed Istio using the operator specs outlined above, this new DNS will be enabled by default.
+
+When running Istio < 1.8.x the following changes add the `.global` stub domain for multicluster communication.
```
Comment (joekelley) [inline docs/content/guides/installing_istio.md:274]: nit: or when using core DNS as opposed to Istio DNS, since Istio DNS rrequires a preview install

**Review by joekelley — APPROVED** [review]

**Pull request merged**